lib/containers/utils.pm

```diff
@@ -179,4 +179,9 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+sub is_sle_container_supported {
+    return 0 if check_os_release('opensuse', 'PRETTY_NAME') && is_sle;
```

Comment:
I don't understand how it could be that `/etc/os-release` contains 'opensuse' and `is_sle` return true

Reply:
I guess it is if host is `opensuse` and container `sle`, then `sle` container is not supported. Maybe we should be over here way more specific.

Reply:
ok now I understand everything :) can you please add some text above this function clearly stating what and why it is checking ?

Reply:
This makes no sense. The `check_os_release()` is checking the local `/etc/os-release`. The `is_sle()` is checking the `VERSION` variable. If we wanna check the container, we should modify the `check_os_release()` and add the `go_to_target` parameter in similar way as it is in `get_os_release()`.

Reply:
see latest changes. without `is_sle`. and feel free to reopen it if it does not work for you